bom/pom.xml

```diff
@@ -50,24 +50,22 @@
     <logback.version>1.2.9</logback.version>
     <micrometer.version>1.7.6</micrometer.version>
     <mongodb-image.name>mongo:4.4</mongodb-image.name>
-    <mongodb-driver.version>4.3.3</mongodb-driver.version>
+    <mongodb-driver.version>4.3.4</mongodb-driver.version>
     <native.image.name>registry.access.redhat.com/ubi8/ubi-minimal:8.4</native.image.name>
-    <netty.version>4.1.68.Final</netty.version>
-    <netty.tcnative.version>2.0.42.Final</netty.tcnative.version>
-    <opentracing.version>0.33.0</opentracing.version>
```

Comment:
The `opentracing.version` property is still used in the hono-core pom.xml.

Reply:
right, we still support embedding the packages into the OSGi bundle ...

Reply:
we will get rid of that with Hono 2.0.0